certified-connectors/AXtension Content Gate/apiDefinition.swagger.json

```diff
@@ -2442,6 +2522,135 @@
         "summary": "Deletes a Content Gate webhook",
         "x-ms-visibility": "internal"
       }
+    },
+    "/api/ContentEntityProperties": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "id"
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "name"
+                  },
+                  "label": {
+                    "type": "string",
+                    "description": "label"
+                  },
+                  "apiName": {
+                    "type": "string",
+                    "description": "apiName"
+                  },
+                  "propertyType": {
+                    "type": "string",
+                    "description": "propertyType"
+                  },
+                  "allowEditOnCreate": {
+                    "type": "boolean",
+                    "description": "allowEditOnCreate"
```

Comment:
@JoepKillaars All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Adjusted swagger file.

Reply:
@JoepKillaars There are few other descriptions having grammatical issues. Please correct all of them in the swagger.